pandas/tseries/frequencies.py

```diff
@@ -609,7 +537,7 @@ def is_unique(self):
     def is_unique_asi8(self):
         return len(self.deltas_asi8) == 1
 
-    def get_freq(self):
+    def get_freq(self):  # noqa:F811  # Disable flake8 false-positive
```

Comment:
huh?

Reply:
For reasons unknown, flake8 was producing incorrect warnings about the function `get_freq` imported above being redefined.  Here and on line 141.

Reply:
same